### DISCUSSION TITLE
Lets Pass Wikitext to $ActionConfirmWidget

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6123] [Ideas]
The https://tiddlywiki.com/#ActionConfirmWidget is very useful widget!
Unfortunately it does not accept wikitext through `$message`

For example this does not work! (''delete'' bold face in message)

```
<$button>
<$action-confirm $message="Do you wish to ''delete'' the caption and tags?">
<$action-deletefield caption tags/>
Delete "caption" and "tags"
</$action-confirm>
</$button>
```

I would suggest to add this feature to 5.2.1

**Comment by pmario** [comment]
There are no possibilities to style those dialogues. They are built into the browser. ... We will need to completely redesign them, to make that happen. 

Chrome browser wants / will remove those dialogues in certain circumstances anyway. There is a discussion here at GitHub.

**Reply by pmario** [reply]
see: 
Chrome intends to drop support for confirm/alert/prompt in cross-origin iframes https://github.com/Jermolene/TiddlyWiki5/issues/5940
and:  Replace tiddler discard confirmation alert with a wikitext dialog with clearer text #6111

**Comment by kookma** [comment]
I assumed this action is the same as `WidgetMessage: tm-notify` where it accepts wikitext!
So there is a limitation!

Thank you!
